participating.md

```diff
@@ -0,0 +1,144 @@
+# Participating in the CircuitPython weekly meeting
+
+Anyone is welcome to participate in the CircuitPython weekly meeting. Please
+read our [Code of Conduct](https://github.com/adafruit-circuitpython-weekly-meeting/blob/master/CODE_OF_CONDUCT.md)
+before participating to help this project stay welcoming.
+
+The meeting is held in the English language and usually lasts between 60 and 90
+minutes.  If you can only attend part of the meeting, that's absolutely fine.
+
+To participate in the meeting, or just attend it live as a listener, join us on
+the [Adafruit Discord](https://adafru.it/discord). There is a _#circuitpython_
+text channel and a _#circuitpython_ voice channel.
+
+The text from the text channel and the audio from the voice channel are
+recorded and uploaded to YouTube and podcast services. Please keep this in
+mind if you plan to participate.
+
+Before speaking in the meeting, ask any Community Moderator to add you to the
+_@CircuitPythonistas_ role. In this role, you will receive a small number of
```

Comment:
It occurs to me that we should reword this to indicate that being added to the role is **required** for speaking. Note that the role is also not camel cased.

```suggestion
If you wish to speak in the meeting, you must ask any Community Moderator to add you to the
_@circuitpythonistas_ role. In this role, you will receive a small number of
```